cmd/helpMessages.go

```diff
@@ -172,6 +172,18 @@ Log in using a VM's user-assigned identity with an Object ID of the service iden
 Log in using a VM's user-assigned identity with a Resource ID of the service identity:
 - azcopy login --identity --identity-resource-id "/subscriptions/<subscriptionId>/resourcegroups/myRG/providers/Microsoft.ManagedIdentity/userAssignedIdentities/myID"
 
+--NOTICE FOR THE FOLLOWING EXAMPLES:--
```

Comment:
Oh, I see. When I said "up by the note about mime types", I had not realized that that's in a different set of constants, for copy not login. I've just pushed a small change (into this PR) two do what I should have suggested (instead of what I actually suggested ;-)

Reply:
Un-resolving to ensure you see this comment Adele.  Pls re-resolve if it sounds OK